src/kafka.go

```diff
@@ -101,14 +101,18 @@ func getBrokerList(arguments *args.ParsedArguments) ([]*connection.Broker, error
 
 		brokers := make([]*connection.Broker, 0, len(metadata.Brokers))
```

Comment:
Would it be worth to add here a log to report the number of discovered brokers?

Reply:
ok, why not.

Reply:
done in https://github.com/newrelic/nri-kafka/pull/177/commits/a6484f207bdbf3506dca44ee9c706f775ac94322